assets/react/v3/@types/index.d.ts

```diff
@@ -6,6 +6,86 @@ declare module '*.jpeg';
 declare module '*.jpg';
 
 declare global {
+<<<<<<< HEAD
```

Comment:
Push with conflict?? :O

Reply:
upss, sorry my bad. But how come git is allowing this to push. No warning :3